webship-path/src/Webship/Path.hs

```diff
@@ -93,6 +94,6 @@ varParse' e d =
     (Op (pure . e))
     (StateT $ \xs -> case xs of [] -> Left PathConsumed; h : t -> flip (,) t <$> d h)
 
-varInt :: Path Int
-varInt =
-  varParse (T.pack . show) (first T.pack . readEither . T.unpack)
+pathInt :: Path Text -> Path Int
+pathInt =
```

Comment:
would this clash with anything if it was just `int`? I think it would look nicer